bpf/lib/nodeport.h

```diff
@@ -637,6 +729,7 @@ static inline int nodeport_lb4(struct __sk_buff *skb, __u32 src_identity)
 		else
 			return ret;
 	}
+	ct_state_new.orig_dport = key.dport;
```

Comment:
Did this slip in or why is it needed? (short comment would be nice)

Reply:
A leftover from the rebase.